### ISSUE TITLE
Is is necessary to pretrain the speaker encoder using GE2E loss on VCTK?

### ISSUE THREAD

**Issue by KinglittleQ** [auspicious3000/autovc#15]
@auspicious3000 Can I train the model directly on VCTK without pretrain the speaker encoder using GE2E loss?

**Comment by auspicious3000** [comment]
Yes. Just use one-hot speaker embedding.

**Comment by KinglittleQ** [comment]
I want to use the speaker encoder instead of one-hot speaker embedding, but pre-training the speaker encoder using GE2E loss will take much time. So I want to train my speaker encoder within the whole model without pre-train it. Would it work?

**Comment by auspicious3000** [comment]
No. But you can train your own speaker encoder instead of using GE2E
